go-controller/pkg/libovsdbops/meter.go

```diff
@@ -51,9 +51,10 @@ func CreateOrUpdateMeterOps(nbClient libovsdbclient.Client, ops []ovsdb.Operatio
 	for _, band := range meterBands {
 		meter.Bands = append(meter.Bands, band.UUID)
 	}
+	// TODO(trozet): evalute if safe to update all fields of the meter
 	opModel := operationModel{
 		Model:          meter,
-		OnModelUpdates: onModelUpdatesAll(),
+		OnModelUpdates: []interface{}{&meter.Bands, &meter.Fair, &meter.Unit},
```

Comment:
Better to have a fields argument and let them be provided by the caller, as we do in some other places?

Reply:
done